.eslintignore

```diff
@@ -1 +1 @@
-dist/*.js
+dist/
```

Comment:
This was causing problems if you are linting after browser transpilation was done as there is other JS files inside dist/lib/.